### ISSUE TITLE
Find out which irradiance field to use

### ISSUE THREAD

**Issue by tomwhite** [openclimatefix/metoffice_ec2#39]
In https://github.com/openclimatefix/predict_pv_yield_nwp we build a baseline model for predicting PV yield from the NWP irradiance field. This uses the GRIB data, where the field is called "Downward short-wave radiation flux" or `dswrf`.

In #34 we use the irradiance data from the AWS feed in NetCDF/Zarr format. The field names are different (see https://github.com/openclimatefix/metoffice_ec2/issues/2#issuecomment-635930273), but `surface_downwelling_shortwave_flux_in_air` seems like it might be the corresponding field.

Is this correct? (Tagging @tv3141 for possible insight here.)

(Of course, in the future we will use the data from the AWS feed in NetCDF/Zarr format, so this problem will go away.)

**Comment by tv3141** [comment]
Yes `surface_downwelling_shortwave_flux_in_air` should be the right variable for this purpose.

From http://cfconventions.org/Data/cf-standard-names/71/build/cf-standard-name-table.html:

surface_downwelling_shortwave_flux_in_air
alias: surface_downwelling_shortwave_flux
The surface called "surface" means the lower boundary of the atmosphere. Downwelling radiation is radiation from above. It does not mean "net downward". The sign convention is that "upwelling" is positive upwards and "downwelling" is positive downwards. The term "shortwave" means shortwave radiation. **Surface downwelling shortwave is the sum of direct and diffuse solar radiation incident on the surface**, and is sometimes called "global radiation". When thought of as being incident on a surface, a radiative flux is sometimes called "irradiance". In addition, it is identical with the quantity measured by a cosine-collector light-meter and sometimes called "vector irradiance". In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics.

**Comment by tomwhite** [comment]
Thanks @tv3141 that's very helpful!
